### PR TITLE
Readme enhancements

### PR DESCRIPTION
**Pull request by miguelsousa**

### CODE REVIEW

**Comment by miguelsousa** [comment]
I had look at https://github.com/adobe-type-tools/afdko/tree/readme-enhancements and the **Installing** sub-section is not formatted quite right.

**Comment by miguelsousa** [comment]
@cjchapman yes please. Sublime Text is not very good that handling RST markup.
Do you have anything to preview it?

**Comment by cjchapman** [comment]
@miguelsousa, I was using this online editor, but it seems to be offline:
http://rst.ninjs.org/
...here's another one like it that I haven't really used:
http://rst.aaroniles.net/
...and here's an OS X tool that I've briefly experimented with (note: click on "Preview" in the status bar at the top of the window to toggle between raw & rendered text):
https://github.com/retext-project/retext

**Comment by frankrolf** [comment]
Would this tool help you with the formatting?
https://github.com/timonwong/OmniMarkupPreviewer
(Installable for SublimeText via PackageControl)

File: README.rst [hunk 1]
```diff
@@ -1,52 +1,119 @@
...
-Please refer to the file `AFDKO-Overview.html <https://rawgit.com/adobe-type-tools/afdko/master/afdko/AFDKO-Overview.html>`__ for a more detailed description of what is in the AFDKO.
 
 Major Changes
-~~~~~~~~~~~~~
+-------------
+
+The AFDKO has been restructured so that it can be installed as a Python package. It now depends on the user's Python interpreter, and no longer contains its own Python interpreter.
 
-The afdko has been restructured so that it can be installed as a Python package. It now depends on the user's Python interpreter, and no longer contains its own Python interpreter.
+In order to do this, two Adobe-owned, non-open source programs were dropped: **IS** and **checkOutlines**. If these turn out to be sorely missed, an installer for them will be added to the old Adobe AFDKO website.  The current intent is to migrate the many tests in checkOutlines to the newer **checkOutlinesUFO** (which does work with OpenType and Type 1 fonts, but currently does only overlap detection and removal, and a few basic path checks).
```
Comment (frankrolf) [inline README.rst]: In order to do … what?
This text probably comes from when the FDK was first open-sourced, the paragraph feels a bit out of context.

File: README.rst [hunk 2]
```diff
@@ -1,52 +1,119 @@
...
+Comments
+~~~~~~~~
+If you have both the FDK from the Adobe AFDKO web page installed, and the new afdko package installed, the commands in the new afdko will take precedence over commands in the older Adobe FDK, as the Python package directory is added at the beginning of the PATH directory list, and the old installer added the Adobe FDK directory to the end of the list.
+
+Note that the PyPI installer will add the new adko package paths to the start of your system PATH environment variable, and this is not undone by the uninstaller. If you want to completely clean up, you will need to change the PATH environment variable to remove the new afdko executable directories. On the Mac, this means editing the line in your login file that sets the PATH variable. On Windows, this means editing the PATH environment variable in the System control panel.
 
-You can also download and install older versions of the AFDKO from the `Adobe afdko home page <http://www.adobe.com/devnet/opentype/afdko.html>`__.
+You can download older versions of the tools from the `Adobe AFDKO homepage`_.
```
Comment (frankrolf) [inline README.rst]: Those downloads include `IS` and the old `checkOutlines`, correct? If so, I would add a note about that here.

File: README.rst (hunk 1 above)
Comment (miguelsousa) [reply README.rst]: It's in reference to the previous paragraph. I'll remove the paragraph break.

Comment (frankrolf) [reply README.rst]: Never mind. I assumed the _In order to do this …_ paragraph was about the original decisions made when the FDK was first open-sourced.

**Comment by miguelsousa** [comment]
@frankrolf OmniMarkupPreviewer is really good. Thanks for the suggestion.

**Pull request merged**